source/chapters/library.rst

```diff
@@ -312,8 +312,8 @@ Tracks can be loaded into a deck in several ways:
   and :guilabel:`Load in Deck 2`, among others. Making either selection will
   load a track into a deck.
 * By :ref:`control-keyboard` to load the selected track in the library track
-  table. Use keyboard :kbd:`Up` and :kbd:`Down` arrow button to select a track, :kbd:`Shift` + :kbd:`Left`
-  to load into Deck 1 and :kbd:`Shift` + :kbd:`Right` to load into Deck 2.
+  table. Use keyboard :kbd:`↑` and :kbd:`↓` arrow button to select a track, :kbd:`Shift` + :kbd:`←`
+  to load into Deck 1 and :kbd:`Shift` + :kbd:`→` to load into Deck 2.
```

Comment:
Please revert that since we agreed to keep the literal key names because there are obviously no icons that work reliably  on all platforms.

Reply:
Did we? I though we just agreed to use the "standard" arrows, not the "fancy" ones?

Reply:
I thought we all agreed to use this particular arrows since it works for all platforms?